data/pages/faq-grantees.mdx

```diff
@@ -94,6 +94,8 @@ That’s basically it![^fn-legal]
 Every grantee must submit a progress report every 90 days. A markdown template
 is provided when we request your first report.
 
+If your grant started in January, for example, the 1st report will be due anytime during the month of April and should cover the work done from January thru March, and so on and so forth. You have a 30-day window to work on the report and we appreciate if you keep them on schedule.
```

Comment:
Nano-nit, maybe shift the example forward by two months to underscore a monthly bracketing rather than a quarterly one.

```suggestion
If your grant started in March, for example, the 1st report will be due anytime during the month of June and should cover the work done from March through May, and so on and so forth. You have a 30-day window to work on the report and we appreciate if you keep them on schedule.
```